CoreFarms.cs

```diff
@@ -139,15 +139,22 @@ public void HonorHall(int goldQuant = 100000000)
         Core.Logger($"Farming {goldQuant} gold using HonorHall Method");
 
         Core.RegisterQuests(3992, 3993);
+        Core.Join("honorhall", "r1", "Left");
         while (!Bot.ShouldExit && Bot.Player.Gold < goldQuant)
         {
-            Core.KillMonster("honorhall", "r1", "Center", "Ice Demon", "Battleground E Opponent Defeated", 10, log: false);
-            Core.KillMonster("honorhall", "r1", "Center", "Ice Demon", "HonorHall Opponent Defeated", 10, log: false);
+            while (!Bot.ShouldExit && Bot.Player.Cell != "r1")
+            {
+                Core.Jump("r1", "Left");
+                Core.Sleep();
+            }
+            foreach (int MonsterMapID in new[] { 1, 2 })
+                while (!Bot.ShouldExit && Core.IsMonsterAlive(MonsterMapID, useMapID: true))
+                    Bot.Combat.Attack(MonsterMapID);
```

Comment:
Not needed

Reply:
ismonsteralive (on 1.2.4) no longer works properly.

Reply:
![image](https://github.com/BrenoHenrike/Scripts/assets/47118168/5fd798c4-ca8e-4a0f-970c-dd81b90b1473)

which requires: `using Skua.Core.Models.Monsters;`

Reply:
this is what we have todo nowadays instead of ismonsteralive.